VacaYAY/BusinessLogicLayer/Exceptions/NotFoundException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLogicLayer.Exceptions
+{
+    public class NotFoundException : Exception
+    {
+        public NotFoundException(int Id) : base($"Entity with Id {Id} was not found.")
```

Comment:
This also. But please pay attention to the magic string.

Proposal: try  to centralize magic string in one class: https://stackoverflow.com/questions/3205549/net-strategies-to-avoid-magic-string